src/citrine/resources/ara_definition.py

```diff
@@ -0,0 +1,78 @@
+from typing import List, Union, Optional
+from uuid import UUID
+
+from citrine._rest.collection import Collection
+from citrine._rest.resource import Resource
+from citrine._serialization import properties
+from citrine._session import Session
+from citrine.ara.columns import Column
+from citrine.ara.rows import Row
+from citrine.ara.variables import Variable
+
+
+class AraDefinition(Resource["AraDefinition"]):
```

Comment:
Will the term "Ara" be user-facing, or will we change this name before release?

Reply:
My expectation is that it will change, but we've released codenames before on a temporary basis.